assets/js/blocks/reviews/editor-block.js

```diff
@@ -52,26 +51,20 @@ class EditorBlock extends Component {
 		}
 
 		return (
-			<BlockErrorBoundary>
```

Comment:
won't it still be needed here for the editor context (`renderFrontend` isn't used in the editor context right?)